public/js/editors/libraries.js

```diff
@@ -681,6 +681,16 @@ var libraries = [
     'url': 'https://cdnjs.cloudflare.com/ajax/libs/mori/0.3.2/mori.js',
     'label': 'mori 0.3.2',
     'group': 'Data structures'
+  },
+  {
+    'url': 'http://cdn.jsdelivr.net/ramda/0.15.1/ramda.min.js',
```

Comment:
All CDN URLs should be served from https when available, as more and more users are switching to https by default, and this won't load.